### ISSUE TITLE
Implementation of throughtputTimer from the Java Version

### ISSUE THREAD

**Issue by PaulDMuir** [abstracta/jmeter-dotnet-dsl#24]
I am not seeing the ability to do a Constant ThroughPut Controller. I see in the Java version there is a [throughputTimer](https://abstracta.github.io/jmeter-java-dsl/guide/#control-throughput). Would we be able to get that implemented in the .Net version?

**Comment by rabelenda** [comment]
Yes. Thank you asking about this. We will consider it for a future release
